tests/monotouch-test/VideoToolbox/VTCompressionSessionTests.cs

```diff
@@ -127,6 +126,9 @@ public void CompressionSessionGetSupportedPropertiesTest ()
 		// This test is (kind of) expected to be null due to as of iOS 8 all supported properties are not meant to be serialized
 		// see CompressionSessionGetSupportedPropertiesTest.
 		[Test]
+#if MONOMAC
+		[Ignore ("Crashes with SIGSEGV when trying to dispose session after calling session.GetSerializableProperties ()")]
```

Comment:
please file a bug report ^ so we do not forget about it. It might be something that affect iOS too (and we just lucky to avoid the crash)